packages/grafana-data/src/utils/csv.ts

```diff
@@ -1,10 +1,13 @@
 // Libraries
 import { defaults } from 'lodash';
+// eslint-disable-next-line import/order
 import Papa, { ParseConfig, Parser, ParseResult } from 'papaparse';
 
 // Types
+import { read, utils } from 'xlsx';
```

Comment:
can we keep this out of `@grafana/data`?  

Perhaps just copy this whole file to core, and we should likely deprecate the CSV/papaparse -- better to keep these libraries out of the data package if possible

Reply:
I moved the sheetjs code out of this file however I'm not sure we can move the whole thing. The readCSV function is used by TableInputCSV component in grafana/ui which is used by the input-datasource. I propose that we deprecate this in a different PR. WDYT? @ryantxu